gsuite_reports_rules/gsuite_brute_force_login.py

```diff
@@ -1,4 +1,6 @@
 from panther_base_helpers import deep_get, gsuite_details_lookup
+
+# pylint: disable=pointless-string-statement
```

Comment:
As long as the string is meant to be here we are good to go 👍

Reply:
I think this string is just as a comment basically. Could swap it to a block comment if we want.